website/docusaurus.config.js

```diff
@@ -574,19 +578,31 @@ module.exports = {
                         to: '/understanding-unleash/unleash-overview',
                     },
                     {
-                        from: '/tutorials/managing-constraints',
+                        from: [
+                            '/tutorials/managing-constraints',
+                            '/topics/managing-constraints',
+                        ],
                         to: '/understanding-unleash/managing-constraints',
                     },
                     {
-                        from: '/tutorials/the-anatomy-of-unleash',
+                        from: [
+                            '/tutorials/the-anatomy-of-unleash',
```

Comment:
Wait. actually: was this page ever under "tutorials"? That feels off to me. The new redirect looks right though. And that actually goes for all the others you've fixed now too. I don't think they were ever under tutorials? 🤔

Reply:
Yeah - it must have been there for at least a while - I got this from Google's 404 report. I don't know where the link is coming from yet, but I wanted to fix this in the meantime.